BootloaderCorePkg/Stage1A/Ia32/Vtf0/Ia16/Real16ToFlat32.asm

```diff
@@ -111,5 +111,7 @@ jumpTo32BitAndLandHere:
     mov     fs, ax
     mov     gs, ax
     mov     ss, ax
+    /* Restore the BIST value to EAX register */
+    movd     eax, mm0
```

Comment:
Today, the code does not handle CPU BIST info yet.   If we want to add code to handle BIST,  I think this location is still a little bit too early to restore it.   A better place is to restore it just before the code jumps into Stage1A  _ModuleEntryPoint.

Reply:
Acknowledged! Yes you are right, I'll put it just before jumping to the Sec Entry.